SomosMasApp/SomosMasApp.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,113 @@
+{
+  "pins" : [
+    {
```

Comment:
why do you use swift package manager if is not the default manager that we used in the app?

Reply:
I dont use cocoapods due to errors while I try to do the integration. Anyway, i investigate how to do it, solved the problem and do PR again using the default manager used in the app.